test/test_pubsub.py

```diff
@@ -51,6 +53,13 @@ async def test_subscribe_multiple_channels(mock_pubsub):
     assert (1, 2, 3) == tuple(mock_pubsub._subscriptions.keys())
 
 
+@pytest.fixture()
+def mock_pubsub_subscriptions(mocker, mock_pubsub):
+    subscriptions_mock = dict({1: mock_pubsub._redis.pubsub()})
+    mocker.patch.object(mock_pubsub, '_subscriptions', subscriptions_mock)
+    return mock_pubsub
+
+
```

Comment:
Please move fixture to `conftest.py`. Please, mind that there is already a fixture named `mock_pubsub_subscriptions()`.

Reply:
Add the docstring